### ISSUE TITLE
Feature Request Monthly Archive

### ISSUE THREAD

**Issue by m2n037** [msaharia/assamtribunearchive#1]
On the first day of every month, the last month's issues can be moved to a folder such as 2017_03_Mar. This can also be archived every six months to a public Dropbox or Google Drive folder to reduce space requirements.

**Comment by msaharia** [comment]
I agree. Actually, I was supposed to put it as an FTP site. But currently code is using Github as a backup for the files which is not suggested. I implemented hurriedly at that time. Will change setup into monthly.
